tests/gziptests.nim

```diff
@@ -43,4 +43,25 @@ proc main() =
                 break
         s.close()
         w.close()
+
+    # read line by line confirming that behavior of atEnd is consistent with standard FileStream
+    block:
+        let fs = newFileStream("files/gzipfiletest.txt")
```

Comment:
2 space indent is Nim's convention

Reply:
It's consistent with the rest of the test file, though. So do you want me to reindent the entire file or leave it as is?

Reply:
Yes, reindent the whole file please.

Reply:
and gzipfiles too if you can

Reply:
Done.